src/libraries/Microsoft.PowerFx.Interpreter/ReflectionFunction.cs

```diff
@@ -452,10 +452,4 @@ public override async Task<FormulaValue> Unwrap(object obj)
             }
         }
     }
-
-    // A function capable of async invokes. 
-    internal interface IAsyncTexlFunction
```

Comment:
>internal [](http://example.com/codeflow?start=4&length=8)

This got moved to Core so that the new Fx.STJ nuget can pass an implementation of ParseJson to the interpreter